wf/ja-jp/content/topics/workflows/connector-builder/capia-authping.htm

```diff
@@ -319,7 +319,7 @@ <h2>このタスクを開始する<br /></h2>
                                                             <p>ツールバーの<span class="uicontrol">［Save（保存）］</span>ボタンをクリックし、フローに<code>_authping</code>という名前を付けます。<code>_authping</code>は予約名であり、完全に一致させる必要があることに注意してください。</p>
                                                         </li>
                                                         <li value="4">
-                                                            <p>フローの説明フィールドに「<code>Connection health check（接続ヘルスチェック）</code>」と入力し、<span class="uicontrol">［Save all data that passes through the Flow（フローを通過するすべてのデータを保存）］</span>ボックスをチェックして<span class="uicontrol">［Save（保存）］</span>をクリックします。</p>
+                                                            <p>フローの説明フィールドに「<code>Connection health check（接続ヘルスチェック）</code>」と入力し、<span class="uicontrol">［Save all data that passes through the Flow（フローを通過するすべてのデータを保存）］</span>ボックスをチェックして<span class="uicontrol">［保存］</span>をクリックします。</p>
```

Comment:
Crud, here's another instance where they dropped the EN portion of the UI control.